stacker/blueprints/vpc.py

```diff
@@ -109,11 +119,13 @@ def create_dhcp_options(self):
         dhcp_options = t.add_resource(ec2.DHCPOptions(
             'DHCPOptions',
             DomainName=domain_name,
-            DomainNameServers=['AmazonProvidedDNS', ]))
+            DomainNameServers=['AmazonProvidedDNS', ],
+            Condition="CreateInternalDomain"))
```

Comment:
We need to remove this condition - no matter what, we need the DHCPOptions.  The domain name will just hold the external zone if no internal zone is provided already, since the default for InternalZone is "".